views/partials/top-level/removal-pilot-privacy.hbs

```diff
@@ -0,0 +1,8 @@
+<main id="removal-pilot-privacy" class="remove-page dashboard clear-header flx flx-col" data-page-label="Removal Pilot Privacy">
```

Comment:
Random Q: What is the `data-page-label` for, and does it need to be localized? (or is it non-user facing?)